modules/ROOT/pages/fullscreen.adoc

```diff
@@ -23,6 +23,8 @@ tinymce.init({
 });
 ----
 
+NOTE: As of {productname} 6.1, toggling fullscreen mode with the `fullscreen` plugin also fires the `ResizeEditor` event.
```

Comment:
IMO this note will distract the reader and should instead be added to the `ResizeEditor` event on this page https://www.tiny.cloud/docs/tinymce/6/events/#editor-core-events and simply stating that it is also fired by toggling fullscreen on 6.1 and up. Or we could add that this event is fired in this plugin in a new `Events` heading on this page, with the fact that this is only on 6.1 and up.

Reply:
To quote my commit message:

> Moved documentation of `ResizeEditor` event firing when `fullscreen` plugin toggles the fullscreen mode.
> 
> It was a NOTE admonition in`fullscreen.adoc`.
> 
> It is now part of the description field for the `ResizeEditor` event in the *Editor core events* table of `events.adoc`. (And is no longer an admonition.)